com.ibm.wala.cast.python.test/data/tf2_test_callbacks2.py

```diff
@@ -0,0 +1,11 @@
+import tensorflow as tf
+
+# Code from https://www.tensorflow.org/api_docs/python/tf/distribute/Strategy#example_usage_2
+strategy = tf.distribute.MirroredStrategy(["GPU:0", "GPU:1"])
+tensor_input = tf.constant(3.0)
+
+@tf.function
+def replica_fn(input):
+  return input*2.0
+
+result = strategy.run(replica_fn, args=(tensor_input,))
```

Comment:
Why are we using a kwarg? Can you use a positional?

Reply:
I copied an exampled directly from the TF docs.

Reply:
Make another test with using just positionals. You can keep this one, just copy it. Thanks.